sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharpExplodedGraph.cs

```diff
@@ -137,6 +137,10 @@ protected override void VisitBinaryBranch(BinaryBranchBlock binaryBranchBlock, E
                     VisitBinaryBranch(binaryBranchBlock, node, ((CatchFilterClauseSyntax)binaryBranchBlock.BranchingNode).FilterExpression);
                     return;
 
+                case SyntaxKind.CaseSwitchLabel when !CasePatternSwitchLabelSyntaxWrapper.IsInstance(binaryBranchBlock.BranchingNode):

```

Comment:
sorry, it's hard to tell what `!CasePatternSwitchLabelSyntaxWrapper.IsInstance(binaryBranchBlock.BranchingNode)` - could you add a brief comment saying why this `when` filter is used?

Reply:
ping